### DISCUSSION TITLE
Hide the entire card, if no events!

### DISCUSSION THREAD

**Discussion by ps915** [totaldebug/atomic-calendar-revive#1586] [Feature Requests]
I am using this card as a birthday reminder. Works fine so far, but is there an option to hide the entire card, if no event is shown?

<img width="454" alt="Bildschirmfoto 2024-10-30 um 17 47 39" src="https://github.com/user-attachments/assets/1428fd33-1e05-4628-a821-6ba11340fedd">

**Comment by Tee86** [comment]
Display it via conditional card?

**Comment by cpetry** [comment]
I tried using conditional card (or visibility settings on card itself) but this won't work on calender entries.
For example: In case the card shows only the entries that occur in the next 5 days, the condition can't be set up like this.
At least I would not know how.

Also this could get quite complex when adding multiple calenders.. the conditional would also need multiple entities to be checked.
Any suggestions?

**Reply by Tee86** [reply]
It's not that complex really...all you need to do is set up a template sensor that will check your calendar. You can set up any time range you want. The state of the sensor will be a simple count of the entities within a specified time frame.
Then, the condition for the conditional card will be based on that sensor. That's pretty much it.

**Comment by marksie1988** [comment]
this has been implemented in the latest release
